### DISCUSSION TITLE
Display file names in table view

### DISCUSSION THREAD

**Discussion by takuy** [directus/directus#9889] [Archive]
Question - I'm setting up a demo for Directus for my team. This particular data model has a "files" field where a user could upload multiple files. Wanted to see if there was a way to list "filename (download)" in the table display, rather than the numerical ID. I think the normal "File" field also only displays either an image, or the raw value ("filename (disk)"?) - any way to get the original filename to appear instead?

![unknown](https://user-images.githubusercontent.com/1631095/142208122-f271ec0c-4a9d-4d2e-b930-0dfe71577187.png)

I also posted this on the Discord and was told to use a "relational values display" but not sure how to accomplish that. The display options for Files field only has "raw value". For file, there's raw value, file, and thumbnail - not exactly what I'm looking for.

**Comment by benhaynes** [comment]
We might need to add the "Relational Values" Display to this data type. @rijkvanzanten ?

**Reply by takuy** [reply]
Should this be submitted as an issue or moved to feature requests/ideas if it's not currently possible?

**Reply by rijkvanzanten** [reply]
@takuy We've enabled support for file/files in the related values display (https://github.com/directus/directus/pull/9989). This'll be included in the next release 👍🏻

**Comment by takuy** [comment]
@rijkvanzanten Awesome! Thank you!
